domains/ajayck.json

```diff
@@ -4,6 +4,7 @@
     "email": "ajaychekavarck@gmail.com"
   },
   "records": {
-    "CNAME": "ajayportfolio-tau.vercel.app"
+    "CNAME": "cname.vercel-dns.com",
+    "TXT": ["vc-domain-verify=ajayck.is-a.dev,lce0854686c931eble33"]
```

Comment:
Invalid Vercel site verification file name. See dos for more information: https://docs.is-a.dev/guides/vercel Reopen PR when fixed.